server/templates/base.html

```diff
@@ -45,7 +45,7 @@
     <div class="container">{% block content %}{% endblock %}</div>
     <div id="footer" class="text-center">
         {% block footer %}
-        &copy; Copyright 2020 by SATNOGS-GDN.
+        &copy; Copyright 2020 by SATNOGS-GDN. {{footer|safe}}
```

Comment:
You pass this value only from ```index``` action. What happens in other actions? ```base``` template is rendered for all actions.

Reply:
Good point. The footer parameter is now optional. I think it's enough to show it in one place, but it can be easily deployed everywhere if you feel that's useful. We just need to add

`footer=get_footer()` every time we call `render_tamplate(...)`.

Reply:
I think that it may be done better without call this function every time.